Documentation/install/upgrade.rst

```diff
@@ -320,6 +320,50 @@ IMPORTANT: Changes required before upgrading to 1.8.0
   .. note:: The ENI IPAM mode automatically derives the native routing CIDR so
             no action is required.
 
+Upgrading from >=1.7.0 to 1.8.y
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+#. Since Cilium 1.5, the conntrack table size parameter
+   ``bpf-ct-global-tcp-max`` was set to the default value ``1000000`` to retain
+   backwards compatibility with previous versions. In Cilium 1.8 the default
+   value is set to 512K by default in order to reduce the agent memory
+   consumption. This means at maximum 512K concurrent TCP connections across all
+   endpoints are supported by default. To prevent from breaking established
+   connections, ``bpf-ct-global-tcp-max`` must be manually set to
+   ``1000000`` (or any other sufficiently large value) before upgrading.
```

Comment:
Ok, my understanding was that only downsizing the map would cause connection breakage. I've now adjusted the section to state that any map size change will cause disrupted connections and also moved this statement into its own paragraph.

Reply:
`This means at maximum 512K concurrent TCP connections across all endpoints are supported by default.` This statement is actually not correct. There's a lot more added to the CT table, e.g. related entries, service entries, etc. So in the worse case in case of BPF NodePort when the backend is local, this means max 128k service connections. :-(

Reply:
@borkmann TIL, thanks for clarifying. Do you think it's ok if I just remove this sentence? I think it would probably be too much detail in the upgrade guide to document the max. number of connections for all the cases.

Reply:
Yes, I would remove it. Doesn't our helm deployment have a 512k CT table size for TCP anyway? Should there be any mentioned of this in the upgrade doc?

Reply:
Yes, the helm deployment already has this number. I think the main concern is about users who changed the default value or aren't using helm deployment.

Reply:
Right, should this be better clarified in the upgrade guide?

Reply:
Agree, I'll rephrase the section to make it clearer.